### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by kr-sarvesh**
please review my grid component

Screenshot (74)
Screenshot (75)

### CODE REVIEW

File: components/grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                &#x3C;/div&#x3E;</code></div>
+              </pre>
+           </div>
+           <h2>Three Columns Grid</h2>
+
+           <div class="grid-container col-3">
+            <div class="primarybg">Three Columns Grid</div>
+            <div class="secondarybg">Three Columns Grid</div>
+            <div class="warningbg">Three Columns Grid</div>
+          </div>
```
Comment (ArpitPatni) [inline components/grid/grid.html:86]: Three Columns are not distinguishable you can use different color to distinguish them or use border color to distinguish them.

**Review by arulk8 — COMMENTED** [review]
I could see many files in the pull request .creating shorter pull request will reduce complexity.

**Review by arulk8 — COMMENTED** [review]
Good to go

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
+@import url('/css/style.css');
+
+body {
+  background-image: none;
+}
+.wrapper .bottom-nav {
+  width: 100%;
+  height: 45px;
```
Comment (arulk8) [inline components/alert/alert.css]: try using rem.

**Review by arulk8 — APPROVED** [review]

**Pull request merged**